functions/index.js

```diff
@@ -52,19 +52,17 @@ const enpUserCv = '/v1/dojo/users/:uid/cv'
 
 app.use(enpLogin, (req, res, next) => {
   if(req.method != "POST") {
-    res.json(405, {
-      error: {
-        code: "method not allowed",
-        message: `the method ${req.method} is not allowed, please try again with a post`
-      }
+    res.status(405).json({
+      error: "method not allowed",
+      "error-description": `the method ${req.method} is not allowed, please try again with a post`
     })
     return;
   }
   const user = req.body.uid;
 
   db.getUser(user, (user_info) => {
     if(user_info["error"] !== undefined) {
-      res.json(401, user_info)
+      res.status(401).json(user_info)
```

Comment:
@NearTox Please add an Error message similar to the one in line 56, should say something like ```not authorized``` so we know the error at quick glance